ee/clickhouse/queries/funnels/base.py

```diff
@@ -299,6 +299,15 @@ def _get_count_columns(self, max_steps: int):
 
         return ", ".join(cols)
 
+    def _get_people_columns(self, max_steps: int):
```

Comment:
can we keep this in `FunnelNew` so we can be 100% sure it's not being leaked elsewhere? (since everything else is new, only thing that requires backwards compatibility is the main funnel class)